libcodechecker/log/option_parser.py

```diff
@@ -489,6 +491,13 @@ def parse_options(args):
             result_map.compile_opts[idx] = opt.replace('"', r'"\"')
 
     result_map.compiler = shlex.split(args)[0]
+
+    #  If the compiler if g++
+    #  we set the language explicitly to c++
```

Comment:
Terminate the comment with a dot. Also, I think this comment is not true because it will match for every string that ends with `++`. I think the comment should reflect that this is a deliberate decision (i.e. to support cross-compilers with a different name).

Reply:
Also don't break the line prematurely. There is a lot of space later on in the previous line.

Reply:
There was supposed to be some sort of a language and architecture detection in either `log_` or the `option_` parser. Or is this still just a `TODO somewhere down the line?

Reply:
architecture and language detection was already implemented in this file in another patch